docs/Generics/chapters/completion.tex

```diff
@@ -742,7 +742,7 @@ \section{Associated Types}\label{critical pairs}
 \arrow[u, Rightarrow, "\ttgp{0}{0}.(\assocsym{P}{A}.\assocsym{Q}{B}\Rightarrow\assocsym{P}{A}.\texttt{B})"']&&
 \end{tikzcd}
 \]
-Note that this was an overlap of the first kind, so rule (7) is now marked \index{left-simplified rule}\textbf{left-simplified}. Thus, rule (*8) completely supercedes rule (7). Rule (*8) survives minimization and maps to the requirement $\ConfReq{\ttgp{0}{0}.[P]A.[Q]B}{R}$, which appears in the generic signature that we output for this protocol extension:
+Note that this was an overlap of the first kind, so rule (7) is now marked \index{left-simplified rule}\textbf{left-simplified}. Thus, rule (*8) completely supersedes rule (7). Rule (*8) survives minimization and maps to the requirement $\ConfReq{\ttgp{0}{0}.[P]A.[Q]B}{R}$, which appears in the generic signature that we output for this protocol extension:
```

Comment:
OED indicates that `supercede` is permissible. It is no longer "standard" but it is still a correct spelling.

Reply:
I'm fine with treating this as a typo. There's no good reason to use non-standard spellings, even if they were previously accepted.